Makefile

```diff
@@ -53,7 +53,9 @@ GOPKGS_SKIP_TESTS  = $(GOMODULE)/pkg/sources/reconciler/ibmmqsource \
                      $(GOMODULE)/pkg/targets/adapter/ibmmqtarget \
                      $(GOMODULE)/pkg/sources/adapter/ibmmqsource \
                      $(GOMODULE)/cmd/ibmmqsource-adapter \
-                     $(GOMODULE)/cmd/ibmmqtarget-adapter
+                     $(GOMODULE)/cmd/ibmmqtarget-adapter \
+                     $(GOMODULE)/cmd/xslttransformation-adapter \
+                     $(GOMODULE)/pkg/flow/adapter/xslttransformation
```

Comment:
If we're skipping tests here, we might be able to remove the `apt-get install libxml2` from the CircleCI config, in the `test` workflow.

Reply:
Ping.

Those lines are no longer necessary:

https://github.com/triggermesh/triggermesh/blob/5911ee64d89d8d5b7d4e7736f1ff183b08f8d9ff/.circleci/config.yml#L38-L41